### ISSUE TITLE
Audio PlayList widget

### ISSUE THREAD

**Issue by paauserrat** [elementor/elementor#15288]
## Prerequisites
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I would like to have an audio playlist widget like the new Video Playlist widget.

**Describe the solution you'd like**
A new widget for audio playlist.

**Describe alternatives you've considered**
Adding multiple SoundCloud widgets or custom code or other plugins.

**Comment by Aviad-Herman** [comment]
Hey @paauserrat 
Will embedding a Spotify playlist solve your problem?

**Comment by fabianenzensperger** [comment]
> Hey @paauserrat
> Will embedding a Spotify playlist solve your problem?


That's the same question as "Will embedding a Youtube playlist solve your problem?" …

I would also find an audio playlist widget very useful. Gladly from several resources and hosted by myself.

**Comment by d19dotca** [comment]
> Spotify playlist

If considering Spotify, I'd also recommend competitors such as Apple Music. Honestly I'd always recommend the top 2-5 players in each segment to be added as sources to start with, I think that'd be ideal for users.

**Comment by paauserrat** [comment]
> Hey @paauserrat
> Will embedding a Spotify playlist solve your problem?

No, it won't solve my issue as I don't know a lot of CSS and HTML to change the design. Having this widget will allow me to do the same as with the new Youtube widget.

**Comment by Aviad-Herman** [comment]
Thanks for the feedback everyone.
I can't promise anything anytime soon but it's great that everything is documented here for future reference.
